src/booking/OfferRequests/OfferRequests.spec.ts

```diff
@@ -108,6 +108,8 @@ describe('OfferRequests', () => {
 
   test('should create an offer request and return the offer request id when `return_offers` is false', async () => {
     const mockResponseWithoutOffer = { ...mockOfferRequest }
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    //@ts-ignore
     delete mockResponseWithoutOffer.offers
```

Comment:
not-blocking, but I would love to find out why we suddenly need this 👀

Reply:
When I tried to run `yarn test` it was complaining about `delete being optional` because `mockResponseWithoutOffer.offers` is required.